src/System.Net.Sockets/src/System/Net/Sockets/SendPacketsElementFlags.Unix.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.Sockets
+{
+    // NOTE: these values are arbitrary. The APIs that consume SendPacketsElement values are not
+    // supported on *nix.
+    internal enum SendPacketsElementFlags
+    {
+        File = 0,
```

Comment:
Personally, I don't think so: the values are arbitrary, as `SendPacketsElement` is not supported natively on *nix. If we decide that it needs to be emulated, these values should still be valid.